### DISCUSSION TITLE
MVC Example (Main) Sample code

### DISCUSSION THREAD

**Discussion by Art666OTS** [MarimerLLC/csla#2989] [Questions]
Hi All,

I am trying to review the MVC Example using CSLA 6.0.0.In VS2022 it complies fine but at run time I get an error.
![cslaer1](https://user-images.githubusercontent.com/38965119/174006878-d2c0ad92-3032-4683-a77e-89bdc17de4ab.PNG)
I modify the name and click the Save button and the code stops here.

![cslaer2](https://user-images.githubusercontent.com/38965119/174007086-3a0d119e-cfc1-44db-a1be-a007bb86e08c.PNG)
 If I click continue in VS I get the list view with the updated value.

![cslaer3](https://user-images.githubusercontent.com/38965119/174007252-9971b288-4626-4831-baee-02cdd10ab6f6.PNG)

Any suggestions

**Comment by rockfordlhotka** [comment]
That's not good! Clearly a bug lurking in there.
